test/verify/check-machines

```diff
@@ -1744,17 +1746,13 @@ class TestMachines(MachineCase, StorageHelpers, NetworkHelpers):
                                                            storage_volume=partition))
 
         if "debian" not in self.machine.image and "ubuntu" not in self.machine.image:
-            # Test create VM with disk of type "network"
             target_iqn = "iqn.2019-09.cockpit.lan"
             self.prepareStorageDeviceOnISCSI(target_iqn)
 
-            cmds = [
-                "virsh pool-define-as --name poolIscsi --type iscsi --source-host 127.0.0.1 --source-dev {0} --target /dev/disk/by-path/".format(target_iqn),
-                "virsh pool-start poolIscsi",
-                "virsh pool-refresh poolIscsi", # pool-start takes too long, libvirt's pool-refresh might not catch all volumes, so we do pool-refresh separately
-            ]
-            self.machine.execute(" && ".join(cmds))
-            self.addCleanup(self.machine.execute, "virsh pool-destroy poolIscsi; virsh pool-delete pool-delete; virsh pool-undefine poolIscsi")
+            print(self.machine.execute("virsh pool-define-as iscsi-pool --type iscsi --target /dev/disk/by-path --source-host 127.0.0.1 --source-dev {0} && ls -la /dev/disk/by-path && virsh pool-start iscsi-pool".format(target_iqn)))
```

Comment:
I would like to keep this debug into master for a while, now it does not fail, but it used to scarcely, and I would like to see the output is it happens again.

Reply:
Ah and I renamed the poolIscsi -> iscsi-pool, that change is not related, we just name iscsi pools like this in the tests elsewhere, and it makes it easier for me to search in the test file.